cmd/manager/main.go

```diff
@@ -128,6 +132,7 @@ func parseConfiguration() Config {
 		"when a Custom Resource is deleted")
 	flag.BoolVar(&config.SubObjectDeletionProtection, subobjectDeletionProtectionFlag, subobjectDeletionProtectionDefault, "Defines if the operator overwrites "+
 		"(and consequently delete) subresources that were not previously created by the operator. "+subobjectDeletionProtectionMessage)
+	flag.IntVar(&config.IndependentSyncPeriod, "independent-sync-period", independentSyncPeriod, "The default time, in minutes,  between reconciliations for independent custom resources. (default 15, minimum 5)")
```

Comment:
nit
To make them change with each other?

```suggestion
	flag.IntVar(&config.IndependentSyncPeriod, "independent-sync-period", independentSyncPeriod, fmt.Sprintf("The default time, in minutes,  between reconciliations for independent custom resources. (default %v, minimum %v)", independentSyncPeriod, 5 * time.Minute))
```